packages/server/src/scim/utils.ts

```diff
@@ -244,3 +299,49 @@ export function convertToScimListResponse<T>(Resources: T[]): ScimListResponse<T
     Resources,
   };
 }
+
+/**
+ * Converts a SCIM patch request to a JSONPatch request.
+ * SCIM PATCH operations are not the same as JSONPatch operations.
+ * SCIM PATCH can omit "path" if the operation is "add" or "replace".
+ * SCIM PATCH "path" values can omit the leading "/".
+ *
+ * See: https://www.rfc-editor.org/rfc/rfc7644#section-3.5.2
+ *
+ * @param scimPatch - The original SCIM patch request.
+ * @returns The converted JSONPatch request.
+ */
+export function convertScimToJsonPatch(scimPatch: ScimPatchRequest): Operation[] {
+  return scimPatch.Operations.flatMap((inputOperation) => {
+    const { op, path, value } = inputOperation;
+
+    if (op !== 'add' && op !== 'remove' && op !== 'replace' && op !== 'copy' && op !== 'move' && op !== 'test') {
+      throw new Error('Invalid SCIM patch: unsupported operation');
+    }
+
+    if (path) {
+      // This is a normal JSONPatch operation
+      if (!path.startsWith('/')) {
+        return { ...inputOperation, path: `/${path}` } as Operation;
+      }
+      return inputOperation as Operation;
```

Comment:
Shouldn't this be an error since a path starting with a `/` is not a valid SCIM patch. I suspect you're trying to not be totally strict to the SCIM spec, but then we're in hybrid middle ground which seems like it'd be harder to reason about.

Reply:
Similarly, we aren't validating that the `"schemas":
      ["urn:ietf:params:scim:api:messages:2.0:PatchOp"]` is included.

Reply:
Hm.  Let me go back and check.  I thought SCIM allowed both.